client/package.json

```diff
@@ -14,10 +14,11 @@
     "babel-plugin-transform-decorators": "^6.24.1",
     "core-js": "^3.3.2",
     "lodash": "^4.17.20",
+    "@mdi/font": "^4.5.95",
     "material-design-icons-iconfont": "^5.0.1",
```

Comment:
Uninstall this :)

Reply:
It's still being utilized in some places i believe. I would prefer to leave this Style related refactoring to the style tickets if possible 😁